doc/source/serve/production-guide/config.md

```diff
@@ -0,0 +1,200 @@
+(serve-in-production-config-file)=
+
+# Serve Config Files (`serve build`)
+
+This section should help you:
+
+- understand the Serve config file format.
+- understand how to generate and update a config file for a Serve application.
+
+This config file can be used with the [serve deploy](serve-in-production-deploying) command CLI or embedded in a [RayService] custom resource in Kubernetes to deploy and update your application in production.
+The file is written in YAML and has the following format:
+
+```yaml
+import_path: ...
+
+runtime_env: ...
+
+deployments:
+
+    - name: ...
+      num_replicas: ...
+      ...
+
+    - name:
+      ...
+
+    ...
+```
+
+The file contains the following fields:
+
+- An `import_path`, which is the path to your top-level bound Serve deployment (or the same path passed to `serve run`). The most minimal config file consists of only an `import_path`.
```

Comment:
Not sure what "bound" means in this context?

Reply:
removed